taxi/forms.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.forms import forms

```

Comment:
There's no need to import `forms` from `django.forms` here, as you're not using it in this file. It's a good practice to remove unused imports to keep the code clean and efficient.